### PR TITLE
Allowing dichotomous rows in `tbl_continuous()`

### LINKED ISSUE #1931: Allow dichotomous variables in `tbl_continuous()` ?
There are many tables for analytic results that are the primary result among all subjects on the first line, followed by the same result stratified by another variable. I think allowing a dichotomous variable in `tbl_continuous()` would help this along.

More generally, adding an overall row to all tables could be useful.... 🤔

### PR DESCRIPTION
**Pull request by ddsjoberg**
**What changes are proposed in this pull request?**
* Updated functions `tlb_continuous()` and `tbl_ard_continuous()` to optionally display a single level of a variable instead of all levels. (#1931)

``` r
library(gtsummary)

lapply(
  c("all_subjects", "trt"),
  \(x) cards::ard_continuous(trial |> dplyr::mutate(all_subjects = 1L), by = all_of(x), variables = age)
) |> 
  cards::bind_ard() |> 
  tbl_ard_continuous(variable = "age", include = c("all_subjects", "trt"), value = all_subjects ~ 1L) |> 
  bold_labels() |> 
  as_kable()
```

| **Characteristic** |    **Overall**    |
|:-------------------|:-----------------:|
| **all_subjects**   | 47.0 (38.0, 57.0) |
| **trt**            |                   |
| Drug A             | 46.0 (37.0, 60.0) |
| Drug B             | 48.0 (39.0, 56.0) |

<sup>Created on 2024-09-18 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

**If there is an GitHub issue associated with this pull request, please provide link.**
closes #1931

--------------------------------------------------------------------------------

Reviewer Checklist (if item does not apply, mark is as complete)

- [x] Ensure all package dependencies are installed: `renv::install()`
- [x] PR branch has pulled the most recent updates from master branch: `usethis::pr_merge_main()`
- [x] If a bug was fixed, a unit test was added.
- [x] Run `pkgdown::build_site()`. Check the R console for errors, and review the rendered website.
- [x] Code coverage is suitable for any new functions/features: `devtools::test_coverage()`
- [x] `usethis::use_spell_check()` runs with no spelling errors in documentation

When the branch is ready to be merged into master:
- [ ] Update `NEWS.md` with the changes from this pull request under the heading "`# gtsummary (development version)`". If there is an issue associated with the pull request, reference it in parentheses at the end update (see `NEWS.md` for examples).
- [ ] Increment the version number using `usethis::use_version(which = "dev")` 
- [ ] Run `usethis::use_spell_check()` again
- [ ] Approve Pull Request
- [ ] Merge the PR. Please use "Squash and merge".

### CODE REVIEW

File: R/tbl_continuous.R [hunk 1]
```diff
@@ -61,9 +69,23 @@ tbl_continuous <- function(data,
   include <- setdiff(include, c(by, variable)) # remove by and variable columns from list vars included
   data <- dplyr::ungroup(data) |> .drop_missing_by_obs(by = by) # styler: off
 
+  # assign types and values
+  cards::process_formula_selectors(data[include], value = value)
+  type <- rep_named(include, list("categorical")) |>
+    utils::modifyList(rep_named(names(compact(value)), list("dichotomous")))
```
Comment (Melkiades) [inline R/tbl_continuous.R:75]: This adds the dichotomous values to type list, right?

Comment (ddsjoberg) [reply R/tbl_continuous.R:75]: yep!

**Review by Melkiades — APPROVED** [review]
Lgtm! Thanks Daniel!!! ^^

**Comment by ddsjoberg** [comment]
Thank you so much @Melkiades !!

**Pull request merged**